ethcore/src/client/client.rs

```diff
@@ -1293,10 +1294,11 @@ impl snapshot::DatabaseRestore for Client {
 		let mut tracedb = self.tracedb.write();
 		self.importer.miner.clear();
 		let db = self.db.write();
-		db.restore(new_db)?;
+		db.key_value().restore(new_db)?;
 
+		// TODO: restore blooms properly
```

Comment:
ah, I forgot to reopen the blooms db, will fix it

Reply:
@debris Fixed? If so remove the comment :wave:

Reply:
not yet, that's the only remaining thing :)

Reply:
done!